src/components/TypesetStyle/TypesetStyle.js

```diff
@@ -13,11 +13,888 @@ import InputRange from './InputRange';
 import StickyContainer from './StickyContainer';
 import TypesetExample from './TypesetExample';
 
-import typeScale from './typeScale';
-import typeSets from './typeSets';
-
 const { prefix } = settings;
 
+const typeScale = {
```

Comment:
We don't want to important any of the v10 type docs to v11. We have v11 updates already on the v11 site so don't include any of this in the PR.